presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriter.java

```diff
@@ -123,7 +123,7 @@ public OrcFileWriter(
     @Override
     public long getWrittenBytes()
     {
-        return orcWriter.getWrittenBytes() + orcWriter.getBufferedBytes();
+        return orcWriter.getWrittenBytes();
     }
```

Comment:
Let's have two interfaces for this. Otherwise, we will have regression for https://github.com/prestodb/presto/pull/10760/files